code-of-conduct-languages/hi.md

```diff
@@ -0,0 +1,30 @@
+## CNCF सामुदायिक आचार संहिता v1.0
+
+### आचार संहिता का योगदानकर्ता
+
+इस परियोजना के योगदानकर्ताओं और अनुरक्षकों के रूप में, और एक खुले और स्वागत करने वाले समुदाय को बढ़ावा देने के हित में, हम उन सभी लोगों का सम्मान करने की प्रतिज्ञा करते हैं जो रिपोर्टिंग issues के माध्यम से योगदान करते हैं, फीचर अनुरोध पोस्ट करते हैं, प्रलेखन अपडेट करते हैं, pull requset या patches, और अन्य गतिविधियों में भाग लेते हैं।
```

Comment:
इस प्रॉजेक्ट के कंट्रीब्यूटर और मेंटेनर के रूप में, और एक कम्युनिटी को बढ़ावा देने के हित में, हम उन सभी लोगों का सम्मान करने की प्रतिज्ञा करते हैं जो रिपोर्टिंग issues के माध्यम से योगदान करते हैं, फीचर request पोस्ट करते हैं, प्रलेखन अपडेट करते हैं, pull requset या patches, और अन्य गतिविधियों में भाग लेते हैं।

Reply:
I think, it should be अनुरक्षको instead of अनुरक्षकों
अनुरक्षकों --> escorts
अनुरक्षको --> maintainers

Reply:
Features can be written as
Features --> सुविधाएँ

Reply:
*Typo
pull requset --> pull request

Reply:
I don't see the word 

Isn't `सुविधाएँ` is more related to availability, for example, `हमारे पास पानी की सुविधाएँ हे।`